tests/connect_redirect/connect_redirect_tests.cpp

```diff
@@ -278,15 +279,20 @@ _validate_audit_map_entry(_In_ const struct bpf_object* object, uint64_t authent
 static void
 _load_and_attach_ebpf_programs(_Outptr_ struct bpf_object** return_object)
 {
+    bpf_object_ptr unique_object;
```

Comment:
suggest keep the name of the variable simply as `object`. For the raw pointer you may name it `raw_object`.

Reply:
Done